roles/confluent.variables/defaults/main.yml

```diff
@@ -245,7 +245,7 @@ zookeeper_ssl_mutual_auth_enabled: "{{ssl_mutual_auth_enabled}}"
 zookeeper_sasl_protocol: "{{sasl_protocol if sasl_protocol == 'kerberos' else 'none'}}"
 
 ### Authentication to put on ZK Server to Server connections. Available options: [mtls, digest, digest_over_tls].
-zookeeper_quorum_authentication_type: "{{ 'mtls' if zookeeper_ssl_enabled and zookeeper_ssl_mutual_auth_enabled else zookeeper_sasl_protocol }}"
+zookeeper_quorum_authentication_type: "{% if zookeeper_ssl_enabled and zookeeper_ssl_mutual_auth_enabled %}mtls{% elif zookeeper_sasl_protocol in ['mtls', 'digest_over_tls'] %}{{zookeeper_sasl_protocol}}{% else %}none{% endif %}"
```

Comment:
`zookeeper_sasl_protocol in ['mtls', 'digest_over_tls']` shouldn't this be ['digest', 'digest_over_tls']?

Reply:
Yes thats my mistake. Infact it should be  zookeeper_sasl_protocol == 'digest' because zookeeper_sasl_protcol can be among none, digest, kerberos. 
I will fix this